interchain-exchange/tutorial/00-tutorial.md

```diff
@@ -577,7 +642,7 @@ Buy order book on the target blokchain is now empty.
 
 In this chapter you will implement the logic for creating order books.
 
-In Cosmos SDK the state is stored in a key-value store. Each order book will be stored under a unique key composed of four values: port ID, channel ID, source denom and target denom. For example, an order book for `mcx` and `vcx` could be stored under `ibcdex-channel-4-mcx-vcx`. Define a function that returns an order book store key.
+In Cosmos SDK the state is stored in a key-value store. Each order book will be stored under a unique key composed of four values: port ID, channel ID, source denom and target denom. For example, an order book for `mcx` (`marscoin`) and `vcx` (`venuscoin`) could be stored under `ibcdex-channel-4-mcx-vcx`. Define a function that returns an order book store key.
```

Comment:
```suggestion
In Cosmos SDK, the state is stored in a key-value store. Each order book is stored under a unique key that is composed of four values: port ID, channel ID, source denom, and target denom. 

For example, an order book for `mcx` (`marscoin`) and `vcx` (`venuscoin`) could be stored under `ibcdex-channel-4-mcx-vcx`. Define a function that returns an order book store key.
```

Reply:
let's focus on present tense, even with tutorials, I know we struggle with this in all of our tutorials, see https://developers.google.com/style/tense